src/App.tsx

```diff
@@ -22,7 +27,43 @@ function App() {
 
   return (
     <>
-      <div>
+      <AppBar position="static">
+        <Toolbar>
+          <Typography variant="h6" color="inherit" component="div">
+            Invoices
+          </Typography>
+
+          <MenuItem>
+            <Typography textAlign="center">New Invoice</Typography>
+          </MenuItem>
+        </Toolbar>
+      </AppBar>
+
+      <InvoiceList></InvoiceList>
+
+      <Invoice></Invoice>
+
+      {/* Pytanie: dlaczego jak daje m={2} dodaje mi poziomego scrolla */}
+      <Grid container spacing={2} p={4} direction={'column'} mb={12}>
+        <Grid item container>
```

Comment:
Why is this needed here? I think something like that:
```        
<Grid container spacing={2} p={4} direction={'column'} mb={12}>
            <Grid
                item
                xs={6}
                container
                justifyContent="center"
                alignItems="flex-start"
                gap={2}
                sx={{ height: 'fit-content' }}
            >
                {/* Kiedy to usuwam to pojawia się błąd w konsoli */}
                <Button variant="contained" startIcon={<SaveIcon />}>
                    Send
                </Button>
            </Grid>
        </Grid>
```
Should have the same effect?